book/animations.md

```diff
@@ -5,56 +5,153 @@ prev: scheduling
 next: skipped
 ...
 
-The UI of a complex web application requires not just fast loading, visually
-interesting rendering, and responsiveness to input and scrolling. It also needs
-smooth *animations* when transitioning between DOM states. These transitions
-improve usability of the web application by helping users understand what
-changed, and improve visual polish by replacing sudden jumps with smooth
-interpolations.
-
-Modern browsers have APIs that enable animating the styles of DOM elements. To
-implement these APIs performantly, behind the scenes GPU acceleration and
-compositing is needed to make those animations smooth and fast.
-
-Types of animations
-===================
-
-An [animation] is a sequence of pictures shown in quick succession that create
-an illusion of *movement* to the human eye.[^general-movement] The pixel
-changes in an animation are *not* arbitrary; they are ones that feel logical to
-a human mind trained by experience in the real world.
-
-[^general-movement]: Here *movement* should be construed broadly to encompass
-all of the kinds of visual changes humans are used to seeing and good at
-recognizing---not just movement from side to side, but growing, shrinking,
-rotating, fading, blurring, and sharpening.
+Complex web application use *animations* when transitioning between
+DOM states. These transitions improve usability by helping users
+understand what changes are occuring. They also improve visual polish
+by replacing sudden jumps with gradual change. But to execute these
+animations smoothly, the browser must make use of the computer's GPU
+and minimize work using compositing.
+
+JavaScript Animations
+=====================
+
+An [animation] is a sequence of still pictures shown in quick
+succession that create an illusion of *movement* to the human
+eye.[^general-movement] Web pages typically animate effects like
+changing color, fading an element in or out, or resizing an element.
+Browsers also use animations in response to user actions like
+scrolling, resizing, and pinch-zoom. And some types of animated edia,
+like videos, can also be included in web pages.[^video-anim] In this
+chapter we'll focus mostly on web page animations, though we'll touch
+on scrolling at the end.[^excuse]
```

Comment:
I'll check and add it back if not.

Reply:
Ah—it was this footnote:

> Video animations is a topic unto itself, but is beyond the scope of this book. Arguably, canvas is a bit different than the other two, since it's implemented by developer scripts. And of course a canvas can have animations within it that look about the same as some DOM animations.

I put a reference into the video-animation bullet to VideoNG instead of this footnote. I've removed the footnote reference for now but can put it and the footnote back if you'd like.